runtime/doc/starting.txt

```diff
@@ -424,7 +426,21 @@ accordingly.  Vim proceeds in this order:
 	-  The file ".exrc"  (for Unix)
 		    "_exrc"  (for Win32)
 
-4. Load the plugin scripts.					*load-plugins*
+4. Enable filetype and indent plugins.
+	This does the same as the commands: >
+		:runtime! filetype.vim
+		:runtime! ftplugin.vim
+		:runtime! indent.vim
+<	This step is skipped if ":filetype ..." was called before now or if
+	the "-u NONE" command line argument was given.
+
+5. Enable syntax highlighting.
+	This does the same as the command: >
+		:runtime! syntax/syntax.vim
+<	This is skipped with "-u NONE" or "--cmd 'syntax off'" as command line
+	arguments.
```

Comment:
Oh, exactly. The thing is, if I do `nvim -u NORC --cmd 'filetype off'` (with this patch), that *won't* disable the filetype detection, because syntax highlighting will pull it in anyways since it's not explicitly disabled by default. I find that a bit counterintuitive if you don't understand what is going on.